utils/workspace.js

```diff
@@ -127,7 +127,14 @@ class Workspace {
     }
 
     // Re-run npm i to make package-lock dirty.
-    child_process.execSync('npm i');
+    child_process.execSync('npm i', {
+      env: {
+        ...process.env,
+        // Playwright would download the browsers because it has e.g. @playwright/browser-chromium or playwright-chromium
+        // in the workspace. We don't want to download browsers here.
+        PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD: '1',
```

Comment:
Note: I'm not mangling with `ELECTRON_SKIP_BINARY_DOWNLOAD` here because I expect its set globally in the workflows which do not use Electron.